data/json/items/gun/223.json

```
@@ -45,6 +45,39 @@
     "min_cycle_recoil": 1350,
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "fs2000",
+    "copy-from": "rifle_semi",
+    "type": "GUN",
+    "name": "FN FS2000",
+    "description": "A sleek bullpup carbine rifle designed by FN Herstal, complete with an integrated sight accessory rail. The forward-firing ejection port and ambidextrous controls make firing comfortable for both left and right-handed shooting.",
+    "weight": "3565 g",
+    "volume": "6688 mL",
+    "price": 125000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": "223",
+    "dispersion": 150,
+    "durability": 7,
```

```suggestion
    "durability": 9,
```

Feel free to bump this way up. This silly gun has a good reputation for reliability.